doc/proxy-clients.rst

```diff
@@ -101,12 +101,24 @@ You can also pass some options to the Varnish client:
   which tags to invalidate when sending invalidation requests to the caching
   proxy. Make sure that your :ref:`Varnish configuration <varnish_tagging>`
   corresponds to the header used here;
+* ``tag_mode`` (ban): Makes it possible to invalidate tags
```

Comment:
the format is inconsistent between the first point and the rest. lets use "(default: ``ban``)"